### ISSUE TITLE
Update KubeRay version to v0.5.0

### ISSUE THREAD

**Issue by anishasthana** [opendatahub-io/distributed-workloads#45]
We are currently installing KubeRay version v0.3.0. The latest available KubeRay is 0.5 -- we should upgrade to this version and confirm that no existing functionality is broken.

**Comment by KPostOffice** [comment]
I'm planning on picking this up. Is working through the existing notebooks on the `codeflare-sdk` repository sufficient? or is there some other definition of "existing functionality" that should be tested?

**Comment by MichaelClifford** [comment]
@KPostOffice, yeah I think making sure the notebooks still work with v5 is probably sufficient.

**Comment by tedhtchang** [comment]
Verify that the cluster.wait_ready returns with v0.5.0.

**Comment by KPostOffice** [comment]
`wait_ready`, `status`, etc. all seem to work as intended

**Comment by KPostOffice** [comment]
If anyone has run into any issues with v0.5.0 that are reproducible. Please LMK

**Comment by anishasthana** [comment]
@KPostOffice do you want to raise a PR upgrading the kuberay version? It seems like the testing went smoothly
